verifier/verifier.go

```diff
@@ -177,6 +179,18 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 			return err
 		}
 
+		pluginVersion := metadata.Version
+		var semVerRegEx = regexp.MustCompile(`^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$`)
+
+		//checking if the plugin version is in valid semver format
+		if !semVerRegEx.MatchString(pluginVersion) {
+			return notation.ErrorVerificationInconclusive{Msg: fmt.Sprintf("plugin %s has pluginVersion %s which is not in valid semver format", verificationPluginName, pluginVersion)}
+		}
+
+		if !isRequiredVerificationPluginVer(pluginVersion, verificationPluginMinVersion) {
```

Comment:
will this throw an error when `verificationPluginMinVersion` field is not present in signature?

Reply:
No, this will treat the other value as higher than this empty value and returns true